ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue

```diff
@@ -232,6 +242,15 @@ export default defineComponent({
       invalidSpaces()
     )
 
+    const emailRules = customRules(
+      isEmail(),
+      invalidSpaces()
+    )
+
+    const phoneRules = customRules(
+      minLength(14),
+      invalidSpaces()
+    )
```

Comment:
Looks good, although we will have a bit of a collision in PR's. 
I moved those function definitions into the components localState as computed properties